ytdlp_handler.py

```diff
@@ -33,12 +34,28 @@ def video_dl(values: Dict) -> None:
         values["AudioOnly"],
         values["path"],
         values["Subtitles"],
+        not values["IsPlaylist"],
         trim_start,
         trim_end,
     )
     with yt_dlp.YoutubeDL(ydl_opts) as ydl:
         infos_ydl = ydl.extract_info(values["url"])
+
     DL_PROGRESS_WINDOW.close()
+
+    if "_type" in infos_ydl.keys() and infos_ydl["_type"] == "playlist":
+        for video_index, infos_ydl_entry in enumerate(infos_ydl["entries"]):
```

Comment:
Update the variable `VIDEO_INDEX` here like this:
`VIDEO_INDEX = video_index + 1`

Reply:
The problem here is theses lines are executed after all downloads. So if I update the global variable `VIDEO_INDEX` there will be no effect on the progress bar. The progress bar will have been closed at this point.

In fact, the progress bar (and download) is closed at the end execution of this line :
`for video_index, infos_ydl_entry in enumerate(infos_ydl["entries"]):`

Or I don't understand the goal of this global variable ?

Reply:
No you're right, it won't work with my suggestion. We need to change the way the progress_bar is initialized

Reply:
I don't know if it's necessary to change the progress bar initialization for now ? 
It seems the `info_dict` (in `d` parameter) contains all needed information for the counter.

Or you wan't have the counter for only one video to download, right ?

Reply:
You're right. We only need to change when we close the progress bar

Reply:
I pushed a commit. It remove the counter when only one video is downloaded (to avoid X/1).

When a playlist contains only one video, the counter doesn't displayed.

You can test to check if this is what you want.